### PR TITLE
Feature/greek localization & Fix for resource loading

### PR DESCRIPTION
**Pull request by alladinian**
Apart from the Greek localization, there is also a fix for loading resources (`.module` instead of `Bundle(for: Type)`) which resolves the issues I (and suppose others) had where language changes did not resolve correctly to the appropriate localized resources.

### CODE REVIEW

File: WeScan/Review/ReviewViewController.swift [hunk 1]
```diff
@@ -27,14 +27,14 @@ final class ReviewViewController: UIViewController {
     }()
     
     private lazy var enhanceButton: UIBarButtonItem = {
-        let image = UIImage(systemName: "wand.and.rays.inverse", named: "enhance", in: Bundle(for: ScannerViewController.self), compatibleWith: nil)
+        let image = UIImage(systemName: "wand.and.rays.inverse", named: "enhance", in: .module, compatibleWith: nil)
```
Comment (wetransferplatform) [inline WeScan/Review/ReviewViewController.swift:30]: - :no_entry_sign: type 'Bundle?' has no member 'module'

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Failed to get the coverage - Error: The data couldn’t be read because it isn’t in the correct format.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Edit/EditScanViewController.swift#L34** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Edit/EditScanViewController.swift#L41** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Edit/EditScanViewController.swift#L75** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Review/ReviewViewController.swift#L37** - type 'Bundle?' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Review/ReviewViewController.swift#L71** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L39** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L46** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L54** - type 'Bundle?' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L136** - type 'Bundle?' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L238** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L241** - type 'Bundle' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L248** - type 'Bundle?' has no member 'module'
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **WeScan/Scan/ScannerViewController.swift#L249** - type 'Bundle?' has no member 'module'
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the WeScanSampleProject editor. (in target 'WeScanSampleProject' from project 'WeScan')</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.1, but the range of supported deployment target versions is 9.0 to 14.3.99. (in target 'FBSnapshotTestCase iOS' from project 'FBSnapshotTestCase')
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **WeScan/Extensions/CIImage+Utils.swift#L16** - 'init(source:)' was deprecated in iOS 12.0: Core Image Kernel Language API deprecated. (Define CI_SILENCE_GL_DEPRECATION to silence these warnings)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **WeScan/Scan/CaptureSessionManager.swift#L266** - 'jpegPhotoDataRepresentation(forJPEGSampleBuffer:previewPhotoSampleBuffer:)' was deprecated in iOS 11.0
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/90dd564a7eb2ea18" target="_blank">Bitrise</a>
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 03fc40d339cd0752574c538ab0114d2debc2eebb
</p>

**Comment by AvdLee** [comment]
@alladinian feel free to reopen with localisation fixes at first only. I guess the module change requires a bit more work!

**Comment by alladinian** [comment]
@AvdLee I was a little sloppy for not realizing that `.module` is only available in packages. The thing is that I did the fix in the first place because I noticed that my localized strings (and eventually any other localization that I've tried) didn't get picked up. But I guess when a find a little free time I'll come up with something!

**Pull request closed without merging**